hooks/mutations/votes/useCommitVotes.ts

```diff
@@ -12,13 +12,16 @@ export function useCommitVotes() {
 
   const { mutate, isLoading } = useMutation(commitVotes, {
     onSuccess: (_data, { formattedVotes }) => {
+      console.log({ _data });
       queryClient.setQueryData<VoteExistsByKeyT>(
         [committedVotesKey, address, roundId],
         (oldCommittedVotes) => {
           const newCommittedVotes = { ...oldCommittedVotes };
 
           for (const { uniqueKey } of formattedVotes) {
-            newCommittedVotes[uniqueKey] = true;
+            if (_data?.transactionHash) {
```

Comment:
the _ with data here makes it ignored (let's you define the param without using it) if you do want to use it, rather remove the _.

Reply:
cool good to know